index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+  <title>My_Portfolio</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <header>
+    <div class="top-bar">
+      <img src="/images/Time Style.png">
+      <img src="/images/Group 4.png">
+    </div>
+    <div class="navigation">
+      <p class="logo">Enoch</p>
+      <a href="#"><img src="/images/Enabled.png"></a>
+    </div>
+    <section class="main">
+      <div class="intro">
+        <p class="para1">Hey There.<br> I'm Enoch</p>
+        <p class="para2">I'm a Software Developer</p>
+        <p class="para3">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+      </div>
+      <div class="icon">
+        <a href="#"><img src="/images/github_icon.png"></a>
+        <a href="#"><img src="/images/angel_list.png"></a>
+        <a href="#"><img src="/images/Linkedin_icon.png"></a>
+        <a href="#"><img src="/images/twitter_icon.png"></a>
+        <a href="#"><img src="/images/medium_icon.png"></a>
```

Comment:
Great job, @TaiwoEnoch It would be much nicer if you consider this project as your portfolio. All links should be active - no dead links. You might have to register for some of these to get valid profile addresses, please take time to sign up. It's part of your tools, as a developer.